app/src/main/AndroidManifest.xml

```diff
@@ -2,16 +2,13 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="cz.covid19cz.erouska">
 
-    <uses-feature
-        android:name="android.hardware.bluetooth_le"
-        android:required="true" />
+    <uses-feature android:name="android.hardware.bluetooth_le" android:required="true" />
+    <uses-feature android:name="android.hardware.bluetooth" />
 
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
```

Comment:
Tohle permission už nepotřebujeme ne?

Reply:
Nebyl jsem si jistý, jestli nebude v požadavcích aktivace Exposure Notification po restartu telefonu, nechal jsem i Broadcast receiver, vyjasníme s řízením.

Reply:
Běh na pozadí řeší Google Play Services, naše appka běžet nemusí. To je na Google API pěkný. Jediný co musí přežít restart je ten scheduled task na stahování dat.